jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
```

Comment:
Actually `andThen` is a `flatMap`. In that case `f` returns an `option` in this case `f` returns just a `'b`.